src/main/java/duke/models/LockerList.java

```diff
@@ -161,6 +165,10 @@ public Locker getLocker(int index) {
         return lockerList.get(index);
     }
 
+    public List<Locker> getAllLockers() {
```

Comment:
can remove this? as it is a duplicate for getLockerList(). Like can just use getLockerList() to get the list of lockers